x-pack/plugins/ingest_manager/server/saved_objects/migrations/to_v7_10_0.ts

```diff
@@ -72,3 +79,16 @@ export const migratePackagePolicyToV7100: SavedObjectMigrationFn<
 
   return packagePolicyDoc;
 };
+
+export const migrateSettingsToV7100: SavedObjectMigrationFn<
+  Exclude<Settings, 'kibana_urls'> & {
+    kibana_url: string;
+  },
+  Settings
+> = (settingsDoc) => {
+  settingsDoc.attributes.kibana_urls = [settingsDoc.attributes.kibana_url];
```

Comment:
is it possible for the old settingsDoc to have `undefined` for `settingsDoc.attributes.kibana_url`? if so, we may need to wrap this line in a conditional so that we don't set `[undefined]` as a value for the new doc

Reply:
It's only possible for there to be no settings if they haven't started kibana.  Settings along with the kibana_url gets created during setup and they cannot remove it through the settings api